drivers/network/dd/dc21x4/media.c

```diff
@@ -471,12 +471,14 @@ MediaInitDefaultMedia(
         }
         else
         {
+            ULONG MiiAdvertising, MiiControl;
             Adapter->DefaultMedia = MEDIA_MII;
 
             switch (MediaNumber)
             {
                 case MEDIA_10T:
                     MiiAdvertising = MII_ADV_10T_HD;
+                    MiiControl = 0;
```

Comment:
@disean is that = 0 assignment fine to you?

Reply:
Yeah

Reply:
Thank you. Resolved.